daemon/daemon.go

```diff
@@ -142,6 +142,10 @@ func (daemon *Daemon) Features() *map[string]bool {
 
 // UsesSnapshotter returns true if feature flag to use containerd snapshotter is enabled
 func (daemon *Daemon) UsesSnapshotter() bool {
+	// DOCKER_INTEGRATION_USE_SNAPSHOTTER is used for integration tests only.
+	if os.Getenv("DOCKER_INTEGRATION_USE_SNAPSHOTTER") != "" {
```

Comment:
Perhaps I should use `TEST_INTEGRATION_`, as we have more of those; let me update